internal/proxy/oauthproxy.go

```diff
@@ -216,6 +216,7 @@ func NewReverseProxy(to *url.URL, config *UpstreamConfig) *httputil.ReverseProxy
 			req.Host = to.Host
 		}
 	}
+	proxy.FlushInterval = 100 * time.Millisecond
```

Comment:
I don't think we should supply this default since this is configurable per upstream.

Reply:
That can be dropped, yeah.